pareto/examples/Membrane_Distillation/MD_PARETO.ipynb

```diff
@@ -0,0 +1,377 @@
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Checkbox(value=False, description='Display State Results')"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Define checkboxes for selecting the results to display\n",
+    "system_checkbox = widgets.Checkbox(value=False, description=\"Display System Results\")\n",
+    "design_checkbox = widgets.Checkbox(value=False, description=\"Display Design Results\")\n",
+    "detailed_cost_checkbox = widgets.Checkbox(\n",
+    "    value=False, description=\"Display detailed cost Results\"\n",
+    ")\n",
+    "state_checkbox = widgets.Checkbox(value=False, description=\"Display State Results\")\n",
+    "\n",
+    "display(system_checkbox, detailed_cost_checkbox, design_checkbox, state_checkbox)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Execute the following cell and then click on the \"Run Simulation\" button to initiate the simulation and optimization of the MD flowsheet. This action triggers the comprehensive analysis of the flowsheet based on your specified inputs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "93a1215dff6e448187b7448b7bd0e236",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Button(description='Run Simulation', style=ButtonStyle())"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "8ae872a5619d47bebe45c08dea6d9779",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Output()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Configuration for the \"Run Simulation\" button and its output handling\n",
+    "run_button = widgets.Button(description=\"Run Simulation\")\n",
+    "output = widgets.Output()\n",
+    "\n",
+    "\n",
+    "def on_run_button_clicked(b):\n",
+    "    with output:\n",
+    "        output.clear_output()\n",
+    "        model = run_simulation(salinity_widget.value, flowrate_widget.value)\n",
+    "        # Display selected results based on checkboxes\n",
+    "        if system_checkbox.value:\n",
+    "            md_flow.display_system(model)\n",
+    "        if design_checkbox.value:\n",
+    "            md_flow.display_design(model)\n",
+    "        if state_checkbox.value:\n",
+    "            md_flow.display_state(model)\n",
+    "        if detailed_cost_checkbox.value:\n",
+    "            md_flow.display_detailed_costs(model)\n",
+    "\n",
+    "\n",
+    "run_button.on_click(on_run_button_clicked)\n",
+    "display(run_button, output)"
+   ]
```

Comment:
The initialization log goes beyond the page boundary which I believe is a minor problem and should not delay merging this PR, but this is a not in case this notebook is used for future demos.

Reply:
@JavalVyas2000 for me, when the output from a Jupyter Notebook cell gets too long, it displays a scroll bar on the right to allow me to scroll through all of the output without it taking up too much of the page, e.g.:
![image](https://github.com/project-pareto/project-pareto/assets/26798258/4ffe2018-eb65-4d83-9fcd-834e786ca6e5)

Do you not see something similar?

Reply:
Apologies. It was my terminal which was making it appear that the text was going beyond the boundary of the page. This is fine.